doc/man1/openssl-rand.pod.in

```diff
@@ -49,10 +49,18 @@ Show the output as a hex string.
 
 =back
 
+=head1 NOTES
```

Comment:
I would like to see fewer "NOTES" sections. This should really be the last paragraph of the DESCRIPTION section.

Reply:
Done.